packages/api/app/Controllers/Http/TechnologyController.js

```diff
@@ -223,10 +223,17 @@ class TechnologyController {
 		try {
 			const { init, commit } = getTransaction();
 			trx = await init();
+			const user = await auth.getUser();
 
 			technology = await Technology.create(data, trx);
 
-			const { users } = request.only(['users']);
+			let { users } = request.only(['users']);
+
+			// if users arent supplied, defaults to the logged in user.
+			if (!users) {
```

Comment:
Se não for especificado usuário, o usuário que fez a requisição será o dono da tecnologia.